examples/apps/presence-tracker/src/view.ts

```diff
@@ -17,10 +17,34 @@ export function renderFocusPresence(focusTracker: FocusTracker, div: HTMLDivElem
 	focusDiv.style.fontSize = "14px";
 
 	const onFocusChanged = () => {
+		const currentUser = focusTracker.audience.getMyself()?.userName;
+		const focusPresences = focusTracker.getFocusPresences();
+
 		focusDiv.innerHTML = `
-            Current user: ${focusTracker.audience.getMyself()?.userName}</br>
+            Current user: ${currentUser}</br>
             ${getFocusPresencesString("</br>", focusTracker)}
         `;
+
+		const existingMessageDiv = document.getElementById("message-div");
+
+		if (existingMessageDiv) {
+			wrapperDiv.removeChild(existingMessageDiv);
```

Comment:
Rather than destroying/creating the div, maybe consider using "display: none" to hide it?